### PR TITLE
Add key labels for colemak layout homerow

### PR DESCRIPTION
**Pull request by afreakk**
Not sure if im doing this correctly,
but i wanted something similar to `-k avy` for colemak layout.

### CODE REVIEW

File: src/map.c [hunk 1]
```diff
@@ -32,6 +35,9 @@ void map_init(label_key_mode_e mode)
         label_keysyms = label_alpha_keysyms;
         map_length = LENGTH_ALPHA;
         break;
+    case LABEL_KEY_MODE_COLEMAK:
+        label_keysyms = label_colemak_keysyms;
+        map_length = LENGTH_COLEMAK;
```
Comment (cornerman) [inline src/map.c:40]: Please add a `break;` here, so we do not make mistakes when adding more key-mode options.

**Review by cornerman — CHANGES_REQUESTED** [review]
Nice, looks good to me. I just added a minor comment.

File: src/map.c (hunk 1 above)
Comment (afreakk) [reply src/map.c:40]: ops, yeah ill fix it

**Review by cornerman — APPROVED** [review]
:+1: Thank you!

**Pull request merged**